geospaas_rest_api/tests.py

```diff
@@ -1,3 +1,76 @@
 from django.test import TestCase
+from django.conf import settings
+from django.test import Client
+from geospaas.vocabularies.models import Platform, Instrument, Parameter
+from geospaas.vocabularies.models import ISOTopicCategory, DataCenter
+from geospaas.catalog.models import *
+import json
 
 # Create your tests here.
+class DatasetURITests2(TestCase):
+
+    fixtures = ["vocabularies", "catalog"]
+
+    def test_api_initial_calls(self):
+
+        c = Client()#self.dataset = Dataset.objects.get(pk=1)
+        
+        response = c.get('//api/')
+        self.assertEqual(response.status_code, 200)
```

Comment:
In each test it is better to compare the response with exact output. Checking status code and `assertIn` is not sufficient.
Tests can be used also for learning how the API (or any other code) functions. And it should be crystal clear what is given as input and what is expected as output.
Please add something like that to each test:
```
self.assertEqual(json.loads(response2.content),
{    "id": 365,
    "geometry": "SRID=4326;POLYGON ((-153.13 34.948, -147.554 0.552, -174.438 -3.507, 174.988 30.162, -153.13 34.948))"}
)
```